Source/TrackableEntities.EF.5/DbContextExtensions.cs

```diff
@@ -207,8 +208,16 @@ public static void ApplyChanges(this InterceptorPool pool, IEnumerable<ITrackabl
                 {
                     // Mark modified properties
                     SetEntityState(context, item, parent, propertyName, EntityState.Unchanged, interceptors);
-                    foreach (var property in item.ModifiedProperties)
-                        context.Entry(item).Property(property).IsModified = true;
+                    var entry = context.Entry(item);
+                    // Exclude properties attributed as NotMapped
+                    var notMappedProps = item.GetType()
```

Comment:
I really believe the solution should be Edm-based rather than reflection-based, but that's what I've come up with as an immediate solution.